dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -73,4 +73,9 @@
             ALTER TABLE workflow ADD CONSTRAINT check_workflow_checkerid_globally_unique CHECK(check_workflow_checker_id_is_unique(checkerid));
         </sql>
     </changeSet>
+    <changeSet author="dyuen (generated)" id="drop_old_accepted_column">
+        <sql>DROP TRIGGER insert_organization_user_sync_status_trigger ON organization_user;</sql>
```

Comment:
I think we can also remove the functions that the triggers use: 
https://github.com/dockstore/dockstore/blob/9ba5a9c3657a5613165750ec9e4ff16f4218f3ee/dockstore-webservice/src/main/resources/migrations.1.14.0.xml#L34
https://github.com/dockstore/dockstore/blob/9ba5a9c3657a5613165750ec9e4ff16f4218f3ee/dockstore-webservice/src/main/resources/migrations.1.14.0.xml#L57